examples/attest/test.bash

```diff
@@ -15,21 +15,22 @@ readonly SBOM_PATH="$7"
 # Launch a registry instance at a random port
 source "${REGISTRY_LAUNCHER}"
 REGISTRY=$(start_registry $TEST_TMPDIR $TEST_TMPDIR/output.log)
+trap "stop_registry ${TEST_TMPDIR}" EXIT
```

Comment:
these don't need to change as they are tests. we can keep them as is as we don't have the ghost problem in our CI.